tests/integration/main.rs

```diff
@@ -15,13 +15,15 @@
 // along with subxt.  If not, see <http://www.gnu.org/licenses/>.
 
 mod codegen;
-mod runtime;
 mod utils;
 
 #[cfg(test)]
 mod client;
 #[cfg(test)]
 mod frame;
 
-pub use runtime::node_runtime;
+pub use test_runtime::{
+    node_runtime,
+    METADATA,
```

Comment:
Where's that used?

Reply:
Ah yes, I thought it would be useful to make the metadata available because I saw that metadata was used, but then I realised that it was used via the subxt Client, and so currently this export isn't helping with anything.